src/transformers/modeling_prophetnet.py

```diff
@@ -0,0 +1,2063 @@
+# coding=utf-8
+# Copyright 2020 The Microsoft Authors and The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch ProphetNet model, ported from ProphetNet repo(fairsequery_states version). """
+
+import copy
+import logging
```

Comment:
logging

Reply:
updated it all other files "lxmert, deberta, fsmt" as well now